packages/tools/generate-images.ts

```diff
@@ -1,7 +1,7 @@
-import { pathExists, readFile, writeFile, copyFile, readdir } from 'fs-extra';
+import { fileExtension } from '@joplin/lib/path-utils';
+import { copyFile, pathExists, readdir, readFile, writeFile } from 'fs-extra';
 import { dirname } from 'path';
 import { execCommand } from './tool-utils';
-import { fileExtension } from '@joplin/lib/path-utils';
 const md5File = require('md5-file');
```

Comment:
Also please restore this. We try to restrict the commits to just what's needed to make reviewing, reverting and merging easier